app/assets/stylesheets/chat.scss

```diff
@@ -1114,6 +1114,23 @@
   color: gray;
   width: max-content;
 }
+
+.mbutton {
+  width: 100%;
+  border: none;
+  text-align: left;
+  background: var(--card-bg);
+}
+
+.ulist {
+  list-style-type: none; /* Remove bullets */
+  padding: 0; /* Remove padding */
+  margin: 0; /* Remove margins */
+}
+.llist:hover {
+  background: var(--theme-container-background-hover, #f5f6f7);
```

Comment:
I think this should use `themeable`  (which helps with older browser fallback).

In a similar vein to this:

https://github.com/forem/forem/blob/d654b666170ca93e27ad0e8f5873aeef1e90d432/app/assets/stylesheets/chat.scss#L77-L81

___

```scss
.llist:hover {
  @include themeable(
    background,
    theme-container-background-hover,
    $light-gray
  );
}
```